context/app/static/js/components/Header/HeaderContent/HeaderContent.jsx

```diff
@@ -27,10 +44,15 @@ function HeaderContent({ anchorRef }) {
       {!shouldDisplayMenu && (
         <>
           <FlexNoWrap>
-            {['Donor', 'Sample', 'Dataset'].map((type) => (
-              <HeaderButton key={type} href={`/search?entity_type[0]=${type}`} component={Link}>
-                {`${type}s`}
-              </HeaderButton>
+            {types.map((type) => (
+              <HeaderType>
+                <div>
+                  <StyledSvgIcon component={type.icon} />
+                </div>
+                <HeaderButton key={type.name} href={`/search?entity_type[0]=${type.name}`} component={Link}>
```

Comment:
As it is now, just the link only includes the text.

@tsliaw do we want the icons to be included in each link (clickable)?

<img width="319" alt="Screen Shot 2023-07-21 at 12 15 17 PM" src="https://github.com/hubmapconsortium/portal-ui/assets/62477388/34a93c93-8cf5-4e6a-9f82-22167e35e4d1">

Reply:
@john-conroy Yes that entire area, including the icons, should be clickable

Reply:
@tsliaw do you want margin between the links? In this implementation padding is used so there wouldn't be any 'un-clickable' space between the links.

Reply:
That's a good point, I think margin might be more appropriate in this case. Going to make that change unless Tiffany says otherwise